sos/plugins/etcd.py

```diff
@@ -23,15 +23,46 @@ class etcd(Plugin, RedHatPlugin):
     """etcd plugin
     """
 
+    packages = ('etcd',)
+
     def setup(self):
-        self.add_copy_spec("/etc/etcd")
+        self.address = self.get_address()
+        self.cmd = 'etcdctl '
+
+        self.add_copy_spec('/etc/etcd')
+
+        subcmds = ['--version',
+                   'member list',
+                   'cluster-health',
+                   'ls --recursive',
+                   ]
+
+        for sub in subcmds:
+            self.add_cmd_output("%s %s" % (self.cmd, sub))
+
```

Comment:
Why not form a list of actual commands from a list comprehension on the `subcms` list, and then pass the whole lot into a single `add_cmd_output()`? I think that would be more readable as well as being more terse.

Also please format the list in the conventional `sos` style (opening `[` on the first line by itself, closing `]` on the last line by itself and list content indented one level further than each).